docs/terraform-provider-build.md

```diff
@@ -47,7 +47,8 @@ https://github.com/ocp-power-automation/ocp4-upi-kvm/blob/master/docs/quickstart
 
 ## On IBM Power Systems
 
-**Terraform** binary on IBM Power is not available for download. You will need to install it from source by running:
+**Terraform** binary on IBM Power is not available for download from Hashicorp website.
+You can download it from the following [link](https://www.power-devops.com/terraform) or you will need to compile it from source by running:
```

Comment:
Better give unicamp link?

Reply:
Do we have it ready  ?

Reply:
Latest from https://oplab9.parqtec.unicamp.br/pub/ppc64el/terraform/ is working for me on RHEL.